particles/Tutorial/Kotlin/README.md

```diff
@@ -58,9 +58,9 @@ arcs_kt_particles(
     srcs = ["HelloWorld.kt"],
 )
 ```
-run the build file by running `./tools/bazelisk build particles/HelloWorld:all` from the root arcs folder.
+run the build file by running `./tools/bazelisk build particles/Tutorial/Kotlin/1_HelloWorld:all` from the root arcs folder.
```

Comment:
I think the current text is correct. Above we say:

```
 It’s easiest if we create a folder to house these files so we don’t lose them. As a matter of convention, particles live in the arcs/particles folder. Go ahead and create a new folder in there called ‘HelloWorld’.
```
So if people are following along with the tutorial, the path should be correct.